### ISSUE TITLE
Positional argument of keyword incorrectly matched as keyword of containing command

### ISSUE THREAD

**Issue by mcneish1** [cheshirekow/cmake_format#66]
```
$ cat CMakeLists.txt
[...]
INSTALL(TARGETS gridDynMain RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime)
[...]
```

Expected:
```
[...]
- INSTALL(TARGETS gridDynMain RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime)
+ install(TARGETS gridDynMain
+         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT
+         runtime)
[...]
```
Got:
```
[...]
- INSTALL(TARGETS gridDynMain RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT runtime)
+ install(TARGETS gridDynMain
+         RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT
+         RUNTIME)
[...]
```

`runtime` is not a keyword if it immediately follows `COMPONENT` (or `EXPORT`/`DESTINATION`, but I didn't check to see if it was capitalized in either of those cases)

**Comment by cheshirekow** [comment]
Yeah this is a known deficiency. What's happening is that `cmake-format` sees the token `runtime` which matches a keyword argument of the `install` command... It thinks this is the start of a new keyword argument, and not a positional argument of `COMPONENT`. All the information regarding the situation is there... In particular `cmake-format` knows that `COMPONENT` expects a single positional argument... the logic just isn't there yet to prioritize consumption the positional before checking for keyword of the containing scope.

**Comment by mcneish1** [comment]
Thanks, that makes sense

**Comment by mcneish1** [comment]
One more:
```
-    add_test(NAME testComponentsQuick COMMAND testComponents --run_test=@quick)
-    add_test(NAME testLibraryQuick COMMAND testLibrary --run_test=@quick)
-    add_test(NAME testSystemQuick COMMAND testSystem --run_test=@quick)
+    add_test(NAME testComponentsQuick COMMAND testComponents --RUN_TEST=@QUICK)
+    add_test(NAME testLibraryQuick COMMAND testLibrary --RUN_TEST=@QUICK)
+    add_test(NAME testSystemQuick COMMAND testSystem --RUN_TEST=@QUICK)
```

run_test/quick aren't even reserved words here, so the uppercasing wouldn't ever be correct. Not sure what's going on with this one

**Comment by cheshirekow** [comment]
Both issues appear to be fixed in `0.4.4`. Actually I don't think I fixed them in my current round of changes, so they are probably fixed in `0.4.3` already. Perhaps update from pip and let me know if you see them with the current `0.4.3`.
